index.html

```diff
@@ -1927,79 +908,48 @@ <h4>Error Responses</h4>
-          <p>
-            A canonical representation serves multiple purposes.
-            It is simplifying the parsing process, enables to identify
-            equivalent TDs by simple string comparisons.
-            Furthermore it allows the use of a simple signing mechanism,
-            such as <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a> or JSON Web Signatures [[RFC7515]] and enables
-            identity checks on encrypted TDs.
-          </p>
-	  <p>
-	     The canonical JSON representation format of a TD adopts the JSON Canonicalization Scheme (JCS) defined by
-	     <a href="https://tools.ietf.org/html/rfc8785>RFC-8785</a> including errata <a href="https://www.rfc-editor.org/errata/eid6292">6292</a>.
-	  </p>
-      </section>
-
-
+    <!-- Security -->
+    <section id="security">
+      <h3>Security</h3>
+      <p class="rfc2119-assertion" id="core-profile-security-1">
+        The following is a list of security schemes which conformant Web Things 
+        MAY use and which conformant Consumers MUST support:
+        <ul>
+          <li>no security</li>
+          <li>Basic Auth</li>
+          <li>Digest</li>
+          <li>Bearer Token</li>
+          <li>Oauth2</li>
+        </ul>
+      </p>
 
-    <!--  Normative References
+      <section>
+        <h4>Recommended Practice</h4>
+        <p>When using the "no security" or "Basic Auth"
+          security schemes it is strongly RECOMMENDED to
+          use transport layer encryption.</p>
+      </section>
+    </section>
 
-		<section>
-			<h2 id="normative-references">Normative references</h2>
-			<p>
-				urn_type = a valid URN as defined in <a
-					href=https://tools.ietf.org/rfc/rfc2141.txt>RFC2141</a>
-			</p>
-			<p>
-				uri_type = a valid URI as defined by <a
-					href=https://tools.ietf.org/rfc/rfc3986.txt>RFC3986</a>
-			</p>
-			<p>
-				identifier = a valid ECMAScript identifier name as defined in
-				chapter 11.6 of <a
-					href=http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf>ECMAScript</a>
-			</p>
-			<p>
-				number = a valid ECMAScript number as defined in chapter 20.1 of <a
-					href=http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf>ECMAScript</a>
-			</p>
-			<p>
-				string = a valid ECMAScript string literal as defined in chapter
-				11.8.4 of <a
-					href=http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf>ECMAScript</a>
-			</p>
-			<p>
-				date = a valid ECMAScript date as defined in chapter '20.3.1.16 Date
-				Time String Format' of <a
-					href=http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf>ECMAScript</a>
-			</p>
-			<p>(https://tools.ietf.org/html/rfc6068).</p>
-		</section -->
-  </section>
-  <section id="open-issues">
-    <h3>Open Issues</h3>
-    <ul>
-      <li>define action semantics</li>
-      <li>define subscription mechanism</li>
-      <li>Examples</li>
-    </ul>
+    <!-- Discovery -->
```

Comment:
I don't think we will be able to define a discovery mechanism in the first profile release.

Reply:
I've been thinking about this and I agree that might be the outcome.

Of the introduction mechanisms defined by the WoT Discovery specification, I think [DNS-based Service Discovery](https://w3c.github.io/wot-discovery/#introduction-dns-sd) could be a candidate for a mandatory discovery mechanism in the Core Profile, but that may be asking too much.

My current thinking is that  [Direct](https://w3c.github.io/wot-discovery/#introduction-direct) discovery could be made the mandatory discovery mechanism by simply requiring that a Thing Description conforming to the Core Profile must be resolvable through an HTTP URL which can be manually typed or copy & pasted in order to locate the Thing Description.